include/deal.II/grid/reference_cell.h

```diff
@@ -434,8 +418,8 @@ namespace ReferenceCell
     AssertIndexRange(n_vertices, 9);
     const auto X = Type::Invalid;
 
-    static constexpr std::array<std::array<ReferenceCell::Type::CellKinds, 9>,
-                                4>
+    static const std::array<std::array<ReferenceCell::Type, 9>,
+                            4>
```

Comment:
This is surprising. Why can't this be constexpr any more?

Reply:
Because the object's constructors are no longer `constexpr` (wheras the constructors for `CellKinds` are: It's just an integer). 

It doesn't matter here because the compiler will simply construct the object on first pass through this function, rather than at compile time. I might try to address this in a follow-up later on as well, but would like to defer this comment to a later patch as well.